connector/protobuf/src/main/scala/org/apache/spark/sql/protobuf/utils/ProtobufUtils.scala

```diff
@@ -145,9 +147,10 @@ private[sql] object ProtobufUtils extends Logging {
    *  the `messageName` is treated as Java class name.
    * @return
    */
-  def buildDescriptor(messageName: String, descFilePathOpt: Option[String]): Descriptor = {
-    descFilePathOpt match {
-      case Some(filePath) => buildDescriptor(descFilePath = filePath, messageName)
+  def buildDescriptor(messageName: String, binaryFileDescriptorSet: Option[Array[Byte]])
```

Comment:
Please fix the scaladoc of this function due to parameter changes

Reply:
Updated.